specs/requests/artifacts_spec.py

```diff
@@ -133,6 +133,73 @@
                 with it('returns a 422 status code'):
                     expect(self.response.status_code).to(equal(422))
 
+        with description('UPDATE MANY'):
+            with context('valid request'):
+                with context("all the resources exist"):
+                    with before.each:
+                        with Mock() as elastic_mock:
+                            for i in range(2):
```

Comment:
you can do: `.get(...).returns(...).times(2)` instead. I think that's a little cleaner.